src/bamands.ts

```diff
@@ -4,35 +4,40 @@ import Discord, { Message } from 'discord.js';
 export async function findMember(
   guild: Discord.Guild,
   ment: string,
-): Promise<string | null> {
-  if (!ment) {
-    return null;
-  }
+): Promise<{ userid: string; error: null } | { error: string; userid: null }> {
   const mention = ment.toLowerCase();
   if (mention.startsWith('<@') && mention.endsWith('>')) {
     let id = mention.slice(2, -1);
     if (id.startsWith('!')) {
       id = id.substr(1);
     }
-    return id;
+    return { userid: id, error: null };
   }
-  const user = guild.member(mention);
-  if (user) {
-    return user.id;
+  // this only works if its not a username, but the ID (NOT a mention)
+  if (/^\d+$/.test(mention)) {
+    const user = await guild.members.fetch(mention);
+    if (user) {
+      return { userid: user.id, error: null };
+    }
```

Comment:
known bug: usernames consisting only of numbers will crash this. IDRC about that right now, but it's noted that this could be a problem at some point.